packages/edit-site/src/components/global-styles/hooks.js

```diff
@@ -215,3 +216,59 @@ export function getSupportedGlobalStylesPanels( name ) {
 
 	return supportKeys;
 }
+
+export function useColorsPerOrigin( name ) {
```

Comment:
I wonder if these would be better on their own "color" file (leaving hooks for the raw hooks only)